samples/apps/copilot-chat-app/deploy/deploy-webapp.ps1

```diff
@@ -89,13 +89,6 @@ if ($LASTEXITCODE -ne 0) {
     exit $LASTEXITCODE
 }
 
-Write-Host "Updating AAD App registration..."
```

Comment:
There might be a way to do this despite the limitations to "az ad app update":
https://stackoverflow.com/questions/68744719/azure-create-app-registration-with-replyurlswithtype

Reply:
Yes, this does seem to work. It overwrites whatever redirectUri is already there, but I think that's ok. I've got it working for PowerShell, doing bash now..